charts/kyverno/Chart.yaml

```diff
@@ -74,3 +74,6 @@ dependencies:
   - name: grafana
     version: "0.0.0"
     condition: grafana.enabled
+  - name: crds
+    version: "0.0.0"
```

Comment:
Do we need to align this version with the release version?

In the main chart we have Helm version `version` and Kyverno version `appVersion`:
>version: 3.1.0-beta.4
appVersion: v1.11.0-beta.4

Reply:
I amn't sure as I found that Prometheus don't align its version. They just use `0.0.0`.